src/options.rs

```diff
@@ -125,21 +147,108 @@ pub struct SessionOptions {
 impl Default for SessionOptions {
     fn default() -> Self {
         Self {
-            datagram_port: 0u16,
-            destination: DestinationKind::Transient,
             nickname: Alphanumeric.sample_string(&mut thread_rng(), 16),
-            publish: true,
+            destination: DestinationKind::Transient,
+            datagram_port: 0u16,
+            datagram_host: 0u128,
+            from_port: 0u16,
+            to_port: 0u16,
+            protocol: 18u8,
+            header: false,
+            inbound_len: 3usize,
+            inbound_qty: 2usize,
+            outbound_len: 3usize,
+            outbound_qty: 2usize,
+            publish_lease_set: true,
+            other_i2cp_options: None,
             samv3_tcp_port: SAMV3_TCP_PORT,
             samv3_udp_port: SAMV3_UDP_PORT,
             silent_forward: false,
-            num_inbound: 2usize,
-            inbound_len: 3usize,
-            num_outbound: 2usize,
-            outbound_len: 3usize,
         }
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct OtherI2CPOptions {
```

Comment:
To remain consistent with the current code style, add line breaks between the options